workflows/templates/ncn-reboot-common.yaml

```
@@ -0,0 +1,305 @@
+                      echo "ERROR platform health check has failed"
+                      exit 1
+                    else
+                      echo "INFO Successfully completed platform health check"
+                    fi
+                    
+                    echo "INFO Checking the status of the slurmctld pods"
+                    result=$(kubectl get  pod -n user -lapp=slurmctld -o jsonpath="{.items[*].status.phase}" 2>&1)
+                    if [[ "$result" != "Running" ]]; then
+                      echo "ERROR $(kubectl get  pod -n user -lapp=slurmctld -o jsonpath="{.items[*].metadata.name}") is not running"
+                      exit 1
+                    else
+                      echo "INFO $(kubectl get  pod -n user -lapp=slurmctld -o jsonpath="{.items[*].metadata.name}") is running"
+                    fi
+                    
+                    echo "INFO Checking the status of the slurmdbd pods"
+                    result=$(kubectl get  pod -n user -lapp=slurmdbd -o jsonpath="{.items[*].status.phase}" 2>&1)
+                    if [[ "$result" != "Running" ]]; then
+                      echo "ERROR $(kubectl get  pod -n user -lapp=slurmdbd -o jsonpath="{.items[*].metadata.name}") is not running"
+                      exit 1
+                    else
+                      echo "INFO $(kubectl get  pod -n user -lapp=slurmdbd -o jsonpath="{.items[*].metadata.name}") is running"
+                    fi
+
+                    echo "INFO Validating BGP peering sessions"
+                    result=$(GOSS_BASE=/opt/cray/tests/install/ncn goss \
+                      -g /opt/cray/tests/install/ncn/tests/goss-switch-bgp-neighbor-aruba-or-mellanox.yaml \
+                      --vars=/opt/cray/tests/install/ncn/vars/variables-ncn.yaml validate 2>&1)
+                    if [[ $? -ne 0 ]]; then
+                      echo "ERROR Validation connection to the BGP neighbors of a switch. run 'canu validate network bgp --verbose' to see the status of all BGP neighbors"
+                      exit 1
+                    else
+                      echo "INFO Successfully Validated connection to the BGP neighbors of a switch"
+                    fi
+                    
+                    echo "INFO Checking if any nodes are in a failed state in CFS"
+                    failed_nodes=$(cray cfs v3 components list --status failed --format json | jq -r '[.components[].id]|join(",")' 2>&1)
+                    if [ -z "$failed_nodes" ]; then
+                      echo "INFO All nodes are in configured state in CFS"
+                    else
+                      echo "ERROR $failed_nodes is/are in failed state in CFS"
+                      exit 1
+                    fi
+
+                    echo "INFO Starting Postgres health check"
+                    result=$(/opt/cray/tests/install/ncn/automated/ncn-postgres-tests 2>&1)
+                    if [[ $? -ne 0 ]]; then
```

`result` isn't used and only the exit value is checked
```suggestion
                    if ! /opt/cray/tests/install/ncn/automated/ncn-postgres-tests >/dev/null 2>&1
                    then
```